src/main.c

```diff
@@ -302,7 +286,24 @@ main (int argc, char *argv[])
                               time_now, time_str_appended, current_waste_num);
 
             if (info_status == 0)
+            {
```

Comment:
Hold on a second, I accidentally removed my last commit when messing around with history 😅. Will push in a few seconds.